spring/src/main/java/org/jdbi/v3/spring/DBIFactoryBean.java

```diff
@@ -23,7 +23,7 @@
 
 import org.jdbi.v3.Jdbi;
 import org.jdbi.v3.spi.JdbiPlugin;
-import org.jdbi.v3.tweak.StatementLocator;
+import org.jdbi.v3.statement.StatementLocator;
```

Comment:
Boy am I glad I haven't gotten to the "remove all the old statement locator stuff" part of the `SqlLocator` task.. :)